cmd/show/contributors.go

```diff
@@ -204,7 +204,7 @@ func (m *ContributorModel) fetchContributorPRs(name string) error {
 	var items []list.Item
 	var mergedPullRequests int
 	for _, pr := range resp.Data {
-		if strings.ToLower(pr.State) == "merged" {
+		if strings.ToLower(pr.PrState) == "merged" {
```

Comment:
Should this be using the `pr_is_merged` flag instead?

Reply:
Ah good catch! Yes, I updated this line to:

```go
if pr.PrIsMerged {
```